packages/components/psammead-podcast-promo/src/components/card-image-wrapper.jsx

```diff
@@ -4,11 +4,15 @@ import {
   GEL_GROUP_2_SCREEN_WIDTH_MIN,
   GEL_GROUP_3_SCREEN_WIDTH_MIN,
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
+  GEL_GROUP_B_MIN_WIDTH,
 } from '@bbc/gel-foundations/breakpoints';
 
 const CardImageWrapper = styled.div`
   flex: 0 0 5.5rem;
   margin: ${GEL_SPACING} 0 0 ${GEL_SPACING};
+  @media (max-width: ${GEL_GROUP_B_MIN_WIDTH}rem) {
+    display: none;
+  }
```

Comment:
We'd want to use the `MIN_WIDTH` constants with `min-width` media queries - mixing and matching can cause weird results when the screen is exactly on that breakpoint

As a general coding style, we do mobile-first, which means the styles for smallest devices are our base styles.

The overriding we do I nmedia queries are for larger screens - that's why almost all our media queries are `min-width` rather than `max-width` 🙂 

```suggestion
  margin: ${GEL_SPACING} 0 0 ${GEL_SPACING};
  display: none;
  @media (min-width: ${GEL_GROUP_B_MIN_WIDTH}rem) {
    display: unset;
  }
```